djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
+
+    if version == 'latest':
```

Comment:
An alternative is to use the default dj-stripe version if `STRIPE_API_VERSION` hasn't been set (at all), use the Stripe account setting if None is specified, and then the specified version otherwise.  That probably makes more sense actually.